h/accounts/events.py

```diff
@@ -0,0 +1,4 @@
+class LoginEvent(object):
+    def __init__(self, request, user):
+        self.request = request
+        self.user = user
```

Comment:
Do we need this and this? https://github.com/hypothesis/h/blob/master/h/events.py#L13-L16